sale_order_line_sequence/README.rst

```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=============================
+Sale Order Line with Sequence
+=============================
+
+Propagates Sales Order line sequence to its Invoices and Stock picking.
+
```

Comment:
Can you also explain you can change see and change the sequence in the sale order line? This is a new feature.